### ISSUE TITLE
关于mask的计算

### ISSUE THREAD

**Issue by wutong4012** [renmada/t5-pegasus-pytorch#18]
您好，我对t5_pegasus的输入输出不是很懂。请问可以解释一下为什么是这么操作么？或者可以告诉我在哪里看源码吗？
prob = model(**cur)[0]
mask = cur['decoder_attention_mask'][:, 1:].reshape(-1).bool()
prob = prob[:, :-1]
prob = prob.reshape((-1, prob.size(-1)))[mask]
labels = cur['decoder_input_ids'][:, 1:].reshape(-1)[mask]
感激不尽~~~
